src/Gambot.Module.Inventory/AddToInventoryResponder.cs

```diff
@@ -9,13 +9,23 @@
 
 namespace Gambot.Module.Inventory
 {
+    readonly struct Trigger
```

Comment:
Due to the way .NET manages memory, `struct` is not really recommended for most cases.  While this is certainly a trivial data structure, it would still be better to use a `class` here.  Otherwise it will be pointlessly (and possibly expensively, thanks to the `Regex`, which is itself a class and thus heap-allocated) copied every time we iterate over the triggers in the `foreach` on line 80.

Reply:
My bad, didn't realize. Will fix tonight.

Reply:
all better